util/src/test/java/com/netcracker/ConsoleHelperTest.java

```diff
@@ -0,0 +1,89 @@
+package com.netcracker;
+
+import org.junit.AfterClass;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.contrib.java.lang.system.TextFromStandardInputStream;
+import org.junit.rules.ExpectedException;
+
+import java.io.*;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Modifier;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.contrib.java.lang.system.TextFromStandardInputStream.emptyStandardInputStream;
+
+public class ConsoleHelperTest {
+    private static final PrintStream SYSTEM_OUT = System.out;
+    @Rule
+    public final TextFromStandardInputStream systemIn = emptyStandardInputStream();
```

Comment:
Actually I don't know why should I use this annotation, but without it these two variables don't do, what I expect, but System.out is a constant, so I'm sure that it shouldn't be any annotation, also when I use @Rule annotation, I should make variable public and non-static